rolling/distribution.yaml

```diff
@@ -5834,6 +5834,16 @@ repositories:
       url: https://github.com/ros-controls/ros2_controllers.git
       version: master
     status: developed
+  ros2_easy_test:
+    doc:
+      type: git
+      url: https://github.com/felixdivo/ros2-easy-test.git
```

Comment:
I actually wanted to add https://ros2-easy-test.readthedocs.io here, but I think only `type: git` is supported?

Reply:
@Timple yes, this registry is specifically for build farm generated docs with rosdoc2 (for ROS 2) and the most supported case is using the source repository as the documentation source, generating and uploading docs to docs.ros.org.

I genuinely thought that there was a "documentation" type for `<url>`s in package.xml but it's not in https://www.ros.org/reps/rep-0149.html#url-multiple, if you set your readthedocs site as the project website it will be linked as the project website on index.ros.org and you can link it from the generated API docs / project readme as well which should help direct people.

Reply:
We'll solve that in the package.xml then. But that's not a blocker for this PR right?

Reply:
This is merged into `main` thanks to @Timple's https://github.com/felixdivo/ros2-easy-test/pull/47